src/public/modules/core/services/formsgSdk.client.factory.js

```diff
@@ -1,9 +0,0 @@
-angular.module('core').factory('FormSgSdk', ['$window', FormSgSdkService])
```

Comment:
don't think we need to migrate this file anyways, in React it can be a hook instead of a service